### ISSUE TITLE
vaon12: VA runtime prints noisy messages

### ISSUE THREAD

**Issue by seungha-yang** [intel/libva#694]
`VA_Win32: Couldn't find a driver in the registry for LUID XXX 0. Using default driver: vaon12` looks noisy. Could we remove it?

cc @sivileri

For instance 
```
[va-d3d] PS C:\Work\gstreamer\build> $env:LIBVA_DRIVERS_PATH="C:\Work\d3d12_va\bin"
[va-d3d] PS C:\Work\gstreamer\build> cd ..
[va-d3d] PS C:\Work\gstreamer> gst-inspect-1.0.exe va
VA_Win32: Couldn't find a driver in the registry for LUID 73260 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 74059 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 73922 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 73260 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 73260 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 73260 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 73260 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 73260 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 74059 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 74059 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 74059 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 74059 0. Using default driver: vaon12
VA_Win32: Couldn't find a driver in the registry for LUID 74059 0. Using default driver: vaon12
Plugin Details:
  Name                     va
  Description              VA-API codecs plugin
  Filename                 C:\Work\GSTREA~1\build\SUBPRO~1\GST-PL~3\sys\va\gstva.dll
  Version                  1.23.0.1
  License                  LGPL
  Source module            gst-plugins-bad
  Documentation            https://gstreamer.freedesktop.org/documentation/va/
  Binary package           GStreamer Bad Plug-ins git
  Origin URL               Unknown package origin

  vaav1dec: VA-API AV1 Decoder in Intel(R) Iris(R) Xe Graphics
  vacompositor: VA-API Video Compositor in Intel(R) Iris(R) Xe Graphics
  vadeinterlace: VA-API Deinterlacer in Intel(R) Iris(R) Xe Graphics
  vadevice1av1dec: VA-API AV1 Decoder in Intel(R) Arc(TM) A350M Graphics
  vadevice1compositor: VA-API Video Compositor in Intel(R) Arc(TM) A350M Graphics
  vadevice1deinterlace: VA-API Deinterlacer in Intel(R) Arc(TM) A350M Graphics
  vadevice1h264dec: VA-API H.264 Decoder in Intel(R) Arc(TM) A350M Graphics
  vadevice1h264enc: VA-API H.264 Encoder in Intel(R) Arc(TM) A350M Graphics
  vadevice1h265dec: VA-API H.265 Decoder in Intel(R) Arc(TM) A350M Graphics
  vadevice1h265enc: VA-API H.265 Encoder in Intel(R) Arc(TM) A350M Graphics
  vadevice1postproc: VA-API Video Postprocessor in Intel(R) Arc(TM) A350M Graphics
  vadevice1vp9dec: VA-API VP9 Decoder in Intel(R) Arc(TM) A350M Graphics
  vah264dec: VA-API H.264 Decoder in Intel(R) Iris(R) Xe Graphics
  vah264enc: VA-API H.264 Encoder in Intel(R) Iris(R) Xe Graphics
  vah265dec: VA-API H.265 Decoder in Intel(R) Iris(R) Xe Graphics
  vah265enc: VA-API H.265 Encoder in Intel(R) Iris(R) Xe Graphics
  vapostproc: VA-API Video Postprocessor in Intel(R) Iris(R) Xe Graphics
  vavp9dec: VA-API VP9 Decoder in Intel(R) Iris(R) Xe Graphics

  18 features:
  +-- 18 elements

[va-d3d] PS C:\Work\gstreamer>
```

**Comment by sivileri** [comment]
Yes, looks like we could have that be only a debug message like this:

![image](https://user-images.githubusercontent.com/82116493/223737742-c3a65602-5d4c-4196-84b8-af8b5cd38b05.png)

Alternatively, we could have that be a VA trace message from `va_win32.c` by adding a method to `va_trace.c` and calling it. However, I don't see any backend specific (X11, DRM, Win32, etc) messages being logged via `va_trace`. @dvrogozh @XinfengZhang what would you advise be the best approach to be consistent with the libva project?

**Comment by dvrogozh** [comment]
Well, there is one here:
https://github.com/intel/libva/blob/97cbc87c94351c62a47d87e776488a9317ad398b/va/x11/va_x11.c#L290

Most interesting part for the trace is tracing runtime parameters flow, not an initialization stage. Maybe that's why there are no many traces in backends. And the one I pointed to above is a runtime trace as well.

So, I would add a trace message. I consider this useful information to know where we've looked for during initialization.

By the way, @sivileri, does va trace work correctly on Windows? did you have a chance to check that?
